lib/modules/platform/bitbucket/index.ts

```diff
@@ -521,7 +521,13 @@ export function massageMarkdown(input: string): string {
     .replace(regEx(/<\/?(details|blockquote)>/g), '')
     .replace(regEx(`\n---\n\n.*?<!-- rebase-check -->.*?\n`), '')
     .replace(regEx(/\]\(\.\.\/pull\//g), '](../../pull-requests/')
-    .replace(regEx(/<!--renovate-(?:debug|config-hash):.*?-->/g), '');
+    .replace(regEx(/<!--renovate-(?:debug|config-hash):.*?-->/g), '')
+    .replace(
+      regEx(
+        / - \[ \] <!-- create-all-rate-limited-prs -->🔐 \*\*Create all rate-limited PRs at once\*\* 🔐/g
```

Comment:
i think there are some more of those boxes. eg rebase all open PR.

Reply:
another checkbox:
- [ ] Click on this checkbox to rebase all open PRs at once

Reply:
i investigated step by step until i found this:
https://github.com/renovatebot/renovate/blob/5cc4260ba7afecda435a45333e017f174a964f2b/lib/modules/platform/utils/read-only-issue-body.ts

Reply:
i will fix accordingly

Reply:
but its not removing the whole text, just the checkbox+comment
so the text \*\*Create all rate-limited PRs at once\*\* 🔐/g will still appear if i do it there

Reply:
so currently we see 
**Click on this checkbox to rebase all open PRs at once**
but it has no checkbox/comment.

which also doesnt make sense to me xD why would we have such a message in bitbucket

Reply:
> i investigated step by step until i found this:
> https://github.com/renovatebot/renovate/blob/5cc4260ba7afecda435a45333e017f174a964f2b/lib/modules/platform/utils/read-only-issue-body.ts

that function should probably updated instead here inside bitbucket 🤔

Reply:
that's right, its not clear why its there, only bitbucket is using it

Reply:
i moved it

Reply:
i think that function will be used by upcoming gerrit platform

Reply:
so please check that PR and revert the move when it's used there

Reply:
so i shouldnt move it at all, i should instead add more to it O.o

Reply:
next time just force push to same PR, we squash anyways